myhpom/templates/myhpom/accounts/choose_network.html

```diff
@@ -0,0 +1,135 @@
+                                id="id_healthcare_network_{{ health_network.id }}"
+                                name="healthcare_network_radio"
+                                value="{{ health_network.id }}"
+                            />
+                            <label
+                                class="custom-control-label nobutton-radio__label"
+                                for="id_healthcare_network_{{ health_network.id }}"
+                            >
+                                {{ health_network.name }}
+                            </label>
+                        </div>
+                    </div>
+                    {% endfor %}
+                </div>
+                {% endfor %}
+
+                <div class="row mt-4">
+                    <div class="col-md-6 col-lg-8">
+                        <div class="form-group">
+                            <label for="id_healthcare_network_full">Complete list of healthcare networks:</label>
+                            <select name="healthcare_network" id="id_healthcare_network_full" class="custom-select">
+                                <option value="null" selected="selected"></option>
+                                {% for priority, networks in health_networks.items %}
+                                    <optgroup label="{{PRIORITY.priority}}">
+                                        {% for health_network in networks %}
+                                        <option id="id_healthcare_network_{{ health_network.id }}" name="healthcare_network_option" value="{{ health_network.id }}">{{ health_network.name }}</option>
+                                        {% endfor %}
+                                    </optgroup>
+                                {% endfor %}
+                            </select>
+                        </div>
+                    </div>
+                    <div class="col-md-6 col-lg-4">
+                        <div class="form-group">
+                            <form name="health_network_form" method="POST">
+                                {% csrf_token %}
+                                <!-- store the selected healthcare_network id in the hidden input -->
+                                <input type="hidden" name="healthcare_network" id="id_healthcare_network" />
+                                <label for="id_custom_provider">Not listed? Write network below:</label>
+                                <input class="form-control" type="text" name="custom_provider" id="id_custom_provider" />                                
+                            </form>
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+            <div class="modal-footer">
+                <!-- TODO: add real URL -->
+                <a class="btn btn-primary" href="" onClick="event.preventDefault(); document.health_network_form.submit();">
+                    Go To Dashboard
+                </a>
```

Comment:
Mockup has "Submit" here, which is important to indicate the data entered will be saved I think. Also, there is supposed to be a "Skip for now" link which would allow users to move forward without doing this step...also important for demo today I think.